migrations/V178__Add_MTO_Common_Milestone_And_Solution.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE mto_common_milestone (
+                            id UUID PRIMARY KEY,
+                            category_name ZERO_STRING NOT NULL,

```

Comment:
I think this should probably just be
```suggestion
                            name ZERO_STRING NOT NULL,
```